packages/events-api/src/adapter.ts

```diff
@@ -88,7 +88,7 @@ export class SlackEventAdapter extends EventEmitter {
    */
   public start(port: number): Promise<http.Server> {
     return this.createServer()
-      .then(server => new Promise((resolve, reject) => {
```

Comment:
[Node 12 build is now failing ❌](https://github.com/slackapi/node-slack-sdk/pull/1249/checks?check_run_id=2717943160)
![image](https://user-images.githubusercontent.com/10026538/120313641-30a56b00-c2d2-11eb-8685-1b4ecb13a19a.png)
The error seems to suggest that there aren't any curly braces when there are ... Seems like `tslint` is struggling to parse the code correctly. I wonder if there's a nice way of resolving this without many ignore comments without replacing `tslint` with `eslint` as per #1074

Reply:
Actually tell a lie, I can just try sticking curly braces immediately after `server => { return new Promise ...`
![image](https://user-images.githubusercontent.com/10026538/120314023-b0cbd080-c2d2-11eb-9dac-f844a0c62ca6.png)

Reply:
>I wonder if there's a nice way of resolving this without many ignore comments without replacing tslint with eslint as per #1074

Yeah...  I am thinking so too.

Reply:
Okay so I think this is sorted now with https://github.com/slackapi/node-slack-sdk/pull/1249/commits/8e37ed7852256d36e6d3017f5a3ea7ac746d298d 🤞

What do you think about the other comment about ignoring the error for `ack` @seratch?

Reply:
@connorads Sorry, I overlooked it 🤦 

>can we instead just ignore it? What do you think about adding an ignore for this @seratch?

Thanks for checking this in details. Ignoring the lint warning sounds fine 👍